### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by DiegoMero**
This pull request is to add al the feature to the project like so: 

 - Create a new Rails app called 'hello-rails-react'.
      - Use the command `rails new -j webpack my_new_rails_app` as described in the previous lesson's tutorial.
 - Verify that the `jsbundling-rails` gem is added to your Gemfile.
 - Initialize your project with Git.
 - Install needed node dependencies as described in the previous lesson's tutorial.
 - Create a `babel.config.js` in the root folder of your project as described in the previous lesson's tutorial.
 - Update `webpack.config.js` to include the babel loader as described in the previous lesson's tutorial.
 - Make sure that your project has a Postgres database set up. 
 - Create a table for storing your messages and create 5 different greetings. 
 - Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
 - Create a static view that will be the root of your app.

---------

- Create the `App` component with react-router and render it in your static view.
- Create the `Greeting` component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
- Display the random greeting in your `Greeting` component.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,110 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project feel free to support!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all my coding partners that help me through the obstacles.
+
```
Comment (t-yanick) [inline README.md:110]: - [ ] Great job working on this project and using the the correct ReadMe template. Unfortunately, the License section is missing. Kindly check the [template](https://github.com/microverseinc/readme-template) and add this section.

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
+  root 'root#index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
```
Comment (t-yanick) [inline config/routes.rb]: - [ ] Kindly deleted this commented line of code and any other commented lines of code in your codebase. We have to always keep our codebase clean and professional

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @DiegoMero , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Required gems and dependencies used
:heavy_check_mark: API endpoint created
:heavy_check_mark: App works as expected

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @DiegoMero,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**